src/components/layout/Shell.tsx

```diff
@@ -29,11 +29,14 @@ export function Shell(props: Props) {
           navbar: "navbar overflow-scroll sm:overflow-visible",
         }}
       >
-        <AppShell.Header className="border-b-[#55595B]" zIndex={102}>
+        <AppShell.Header
+          zIndex={102}
+          bg="#2B2F32"
+          className="border-b-[#2B2F32]"
```

Comment:
I couldn't find this change noted [in the doc](https://www.notion.so/metabase/Mobile-Design-Pairing-with-Alessio-11569354c90180559b6be88aaee6539b), but it makes sense that the border is the same color as the background. But maybe `border: none` or `border-color: transparent`, which are simpler, would do the trick with little to none visual difference.

Reply:
@WiNloSt Let me try with `border-none`, thanks! This change was because of a visual artifact of adding the site switcher - there is a tiny grey border separating the navbar and the site switcher on mobile viewports.

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/afe401bba8034afa946b57054e6aab40fb2fd494